From: Diederik de Haas <didi.debian@cknow.org>
Date: Sun, 22 Sep 2024 10:48:22 +0200
Subject: [PATCH] dri: Add Rockchip EBC to kmsro drivers
Forwarded: https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/23575

The Rockchip EBC driver is used in the Pine64 PineNote.

Link: https://lore.kernel.org/linux-arm-kernel/20220413221916.50995-1-samuel@sholland.org/
---
 src/gallium/targets/dri/dri_target.c   | 3 +++
 src/gallium/targets/dril/dril_target.c | 1 +
 src/gallium/targets/dril/meson.build   | 1 +
 3 files changed, 5 insertions(+)

diff --git a/src/gallium/targets/dril/dril_target.c b/src/gallium/targets/dril/dril_target.c
index a94ab09858f..2e628ae1c02 100644
--- a/src/gallium/targets/dril/dril_target.c
+++ b/src/gallium/targets/dril/dril_target.c
@@ -599,6 +599,7 @@ DEFINE_LOADER_DRM_ENTRYPOINT(pl111)
 DEFINE_LOADER_DRM_ENTRYPOINT(rcar_du)
 DEFINE_LOADER_DRM_ENTRYPOINT(repaper)
 DEFINE_LOADER_DRM_ENTRYPOINT(rockchip)
+DEFINE_LOADER_DRM_ENTRYPOINT(rockchip_ebc)
 DEFINE_LOADER_DRM_ENTRYPOINT(rzg2l_du)
 DEFINE_LOADER_DRM_ENTRYPOINT(ssd130x)
 DEFINE_LOADER_DRM_ENTRYPOINT(st7586)
diff --git a/src/gallium/targets/dril/meson.build b/src/gallium/targets/dril/meson.build
index 55f10dc78052..bc972a0c9666 100644
--- a/src/gallium/targets/dril/meson.build
+++ b/src/gallium/targets/dril/meson.build
@@ -90,6 +90,7 @@ foreach d : [[with_gallium_kmsro, [
                'rcar-du',
                'repaper',
                'rockchip',
+               'rockchip-ebc',
                'rzg2l-du',
                'ssd130x',
                'st7586',
-- 
2.45.2

